lib/menu.rb

```diff
@@ -1,7 +1,7 @@
 class Menu 
   attr_reader :dishes
   def initialize
-    @dishes = {}
+    @dishes = []
```

Comment:
How did you come to this conclusion?

Reply:
I initially used one big hash for the entire menu, but then it felt overly complicated having to first index the array by key, it also made adding menu items more difficult as I had to hard code menu item numbers. The array implementation makes it more scalable. If, for example, you wanted to make a method that adds or removes dishes to the menu, it is simple and easy to implement through array manipulation.

Using an array with hashes made more sense to me because I could use array indexes as menu numbers, filling it with hashes with key : value as dish : price.